app/static/js/index.js

```diff
@@ -26,9 +26,9 @@ function start_fermentation(uid) {
         },
     });
 
-};
+}
```

Comment:
why remove the ; here?

Reply:
Sorry, no good reason, I'll restore it. I was in there copy pasting stuff and made it consistent with the others (it's not needed).